Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:lts-stretch
+ARG JAVA_LIB_VERSION=1.29.0
+ARG PROTOC_VERSION=3.11.4
+RUN npm install ts-protoc-gen grpc-tools grpc
```

Or

```suggestion
RUN npm install --gobal ts-protoc-gen grpc-tools
```

Falls das hier geht:
```
ENV PATH="(npm bin ts-protoc-gen --global):${PATH}"
ENV PATH="(npm bin grpc-tools --global):${PATH}"
```

---

not working, agreed with @bobaaaaa to leave it as it is